opencti-platform/opencti-front/src/private/components/settings/organizations/SettingsOrganizationDetails.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FunctionComponent } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import makeStyles from '@mui/styles/makeStyles';
+import Typography from '@mui/material/Typography';
+import Paper from '@mui/material/Paper';
+import Grid from '@mui/material/Grid';
+import Chip from '@mui/material/Chip';
+import { Theme } from '../../../../components/Theme';
+import { useFormatter } from '../../../../components/i18n';
+import { SettingsOrganizationDetails_organization$key } from './__generated__/SettingsOrganizationDetails_organization.graphql';
+import ExpandableMarkdown from '../../../../components/ExpandableMarkdown';
+import MarkdownDisplay from '../../../../components/MarkdownDisplay';
+
+const styles = makeStyles<Theme>((theme) => ({
+  paper: {
+    height: '100%',
+    minHeight: '100%',
+    margin: '10px 0 0 0',
+    padding: '15px',
+    borderRadius: 6,
+  },
+  chip: {
+    fontSize: 12,
+    height: 25,
+    marginRight: 7,
+    textTransform: 'uppercase',
+    borderRadius: '0',
+    width: 150,
+    backgroundColor: theme.palette.background.accent,
+    color: theme.palette.text?.primary,
+  },
+}));
+
+const SettingsOrganizationDetailsFragment = graphql`
+      fragment SettingsOrganizationDetails_organization on Organization {
+        id
+        description
+        contact_information
+        x_opencti_organization_type
+      }
+    `;
+
+interface SettingsOrganizationDetailsProps {
+  settingsOrganizationFragment: SettingsOrganizationDetails_organization$key;
```

Comment:
this can be misunderstood
What do you think of : settingsOrganizationData or settingsOrganizationKey or node

Reply:
fixed!